<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdrawalFailed</class>
 <widget class="QDialog" name="withdrawalFailed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(110,110,110);
text-transform: uppercase;
background-color: rgb(135, 203, 185);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>701</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);
letter-spacing: 2px;
background-color: rgb(246, 186, 111);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Kate ei riit채&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nappi_kirjaudu_ulos">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>550</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242,242,242);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>KIRJAUDU ULOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nappi_muuta_summaa">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242,242,242);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>MUUTA SUMMAA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_saldo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 234, 227);
color: rgb(110,110,110);
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;K채ytett채viss채 xxxx euroa.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
